dev-python/ldap3/ldap3-2.4.1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="A strictly RFC 4511 conforming LDAP V3 pure Python client"
+HOMEPAGE="https://github.com/cannatag/ldap3 https://pypi.python.org/pypi/ldap3"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+# The tests are a mess, and the test config is not included in the
+# PyPi tarball (see https://travis-ci.org/cannatag/ldap3 and
+# https://github.com/cannatag/ldap3/blob/master/test/config.py).
+RESTRICT="test"
+
+RDEPEND=">=dev-python/pyasn1-0.1.8[${PYTHON_USEDEP}]"
```

Comment:
This version of pyasn1 doesn't exist anymore. The latest version available in the main repository. This condition is basically useless. That's OK, I'll fix this line before merging your contribution. Looks good otherwise.

Reply:
Alright. And yes, I'll be checking for this kind of things when copy-pasting ebuilds from previous versions.